jsonrpc/params.go

```diff
@@ -92,6 +98,22 @@ func (p Params) UnmarshalInto(receivers ...interface{}) error {
 		return errors.New("not enough params to decode")
```

Comment:
Is this check required anymore? if you pass more receivers than params, isn't it enough to set them to `nil`? That seems to be the intended behavior copied across from go-ethereum library at line 169

Reply:
not sure to understand